docs/getting-started/security-ui.asciidoc

```diff
@@ -110,3 +110,50 @@ The Administration page allows you to view and manage hosts that are running End
 
 [role="screenshot"]
 image::images/admin-ui.png[]
+
+[discrete]
+[[timeline-accessibility-features]]
+== Accessibility features
+
+Accessibility features, such as keyboard focus and screen reader support, are built into the Elastic Security UI. These features offer additional ways for you to navigate the UI and interact with the application.
+
+[discrete]
+[[draggable-timeline-elements]]
+=== Interact with draggable elements
+
+Use your keyboard to interact with draggable elements in the Elastic Security UI:
+
+* Select a draggable element to give it keyboard focus.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-keyboard-focus.gif[width=100%][height=100%][Demo that shows how to give a draggable element keyboard focus]
+
+* Press *Enter* on an element with keyboard focus to display its menu.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-display-menu.gif[width=100%][height=100%][Demo that shows how to display an element's menu]
+
+* Press the spacebar on your keyboard to begin dragging a field to a different location and the directional keys to move it to a drop area. Press the spacebar again to drop the field.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-drag-and-drop.gif[width=100%][height=100%][Demo that shows how to drag and drop an element to another location in the Elastic Security UI]
+
+[discrete]
+[[timeline-tab]]
+=== Navigate the Elastic Security UI
+Use your keyboard to navigate through rows, columns, and menu options of elements in the Elastic Security UI:
+
+* Use the directional arrows to move keyboard focus right, left, up, and down.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-directional-arrows.gif[width=100%][height=100%][Demo that shows how to drag and drop an element to another location in the Elastic Security UI]
+
+* Use *CTRL + Home* to shift keyboard focus to the first cell in a row. Likewise, use *CTRL + End* to move keyboard focus to the last cell in the row.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-shift-keyboard-focus.gif[width=100%][height=100%][Demo that shows how to drag and drop an element to another location in the Elastic Security UI]
+
+* Use the *Page Up* and *Page Down* keys to scroll through the page.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-page-up-and-down.gif[width=100%][height=100%][Demo that shows how to drag and drop an element to another location in the Elastic Security UI]
```

Comment:
Apologies for not documenting this in https://github.com/elastic/kibana/pull/85767, but would you be willing to add something like the following text + animated gif?

- When a row has an event renderer, like the one pictured below, press `shift` + down arrow to focus the event renderer.
  - Press `tab` or `shift` + `tab` to navigate between fields in an event renderer.
  - Press the up arrow to return to the cells in the current row, or the down arrow to advance to the next row.

![navigate-event-renderer-with-keyboard](https://user-images.githubusercontent.com/4459398/111375602-101b2c80-8664-11eb-9191-bb3a3fdc86ea.gif)